api/envoy/config/trace/v2/trace.proto

```diff
@@ -80,9 +81,28 @@ message ZipkinConfig {
   // trace instance. The default value is false, which will result in a 64 bit trace id being used.
   bool trace_id_128bit = 3;
 
-  // Determines whether client and server spans will shared the same span id.
+  // Determines whether client and server spans will share the same span context.
   // The default value is true.
   google.protobuf.BoolValue shared_span_context = 4;
+
+  // Available Zipkin collector endpoint versions.
+  enum CollectorEndpointVersion {
+    // Zipkin API v1, JSON over HTTP.
+    HTTP_JSON_V1 = 0 [deprecated = true];
```

Comment:
@htuch I feel like we need to maintain the backward compatibility with the existing installation, i.e. not specifying it and default to v1 JSON. But, if you think this can be removed, I'll happy to do it.

Reply:
So some context from the Zipkin side of things...

In Zipkin pretty much everyone has long switched over to V2 model. Especially since the information stored by Zipkin for the last couple of years has been V2 and thus doing a V1 -> V2 conversion at ingestion. 

My preference would be to say that the 0 value defaults to V2 and we explicitly list the various options. We can then use the deprecated label on the explicit value used for `HTTP_JSON_V1`

Reply:
@htuch as discussed offline, I've added a comment block.